File: GROUP_22_2022510137_2023510064_2022510124.pl
use strict;
use warnings;

my @point1 = (6, 148, 72, 35, 0, 33.6, 0.627, 50);
my @point2 = (1, 85, 66, 29, 0, 26.6, 0.351, 31);

my $sum = 0;

for (my $i = 0; $i < scalar(@point1); $i++) {
    my $difference = $point1[$i] - $point2[$i];
    $sum += $difference * $difference;
}

my $distance = sqrt($sum);

print "Euclidean Distance: $distance\n";

File: GROUP_22_2022510137_2023510064_2022510124.fs
let point1 = [| 6.0; 148.0; 72.0; 35.0; 0.0; 33.6; 0.627; 50.0 |]
let point2 = [| 1.0; 85.0; 66.0; 29.0; 0.0; 26.6; 0.351; 31.0 |]

let mutable sum = 0.0

for i in 0 .. point1.Length - 1 do
    let difference = point1.[i] - point2.[i]
    sum <- sum + difference * difference

let distance = sqrt sum

printfn "Euclidean Distance: %f" distance